.murdock

```diff
@@ -99,7 +101,7 @@ compile() {
     test -d ${BINDIR} && echo "-- build directory size: $(du -sh ${BINDIR} | cut -f1)"
 
     # cleanup
-    rm -Rf build
+    rm -rf ${BINDIR}
```

Comment:
there's a `test` tastement above. You should move the `rm` up there and concatenate with `&&`.

Reply:
I've made it a full `if...then`, otherwise that line would become a bit long.